packages/plugin-dom-editable/src/EventNormalizer.ts

```diff
@@ -528,9 +539,13 @@ export class EventNormalizer {
     destroy(): void {
         this._mutationNormalizer.destroy();
         this._unbindEvents();
-        this._shadowNormalizers.forEach(
-            eventNormalizer => eventNormalizer !== this && eventNormalizer.destroy(),
-        );
+        const eventNormalisers = [...this._shadowNormalizers.values()];
+        this._shadowNormalizers.clear();
+        for (const eventNormalizer of eventNormalisers) {
+            if (eventNormalizer !== this) {
```

Comment:
Comment tu pourrais être dans tes propres shadowNormalizers ?

Reply:
this._shadowNormalizers.set(root, this); <= dans le contructeur